app/controllers/profiles_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 # Controller for the user profile page
 class ProfilesController < Profiles::ApplicationController
-  before_action :set_user
+  before_action :set_user, :current_page
```

Comment:
This before_action can be completely removed. The set_user was previously there but is already in `profiles/application_controller`

Reply:
Please see a3b41c47ed02575f26cc4d4f67c869ad194272ff.